boomkat_importer.user.js

```diff
@@ -28,16 +21,18 @@ $(document).ready(function () {
     // selected. We need a timeout here due to a slight delay between
     // clicking the tab and the tab's classes (and the tracklist)
     // being updated.
-    $('ul.tabs li.tab-title a').click(function () {
-        setTimeout(function () {
-            release = retrieveReleaseInfo(release_url);
-            updateLink(release, release_url, false);
-        }, 150);
+    document.querySelectorAll('ul.tabs li.tab-title a').forEach(function (elem) {
+        elem.addEventListener('click', function () {
+            setTimeout(function () {
+                release = retrieveReleaseInfo();
+                updateLink(release, release, false);
```

Comment:
```suggestion
                updateLink(release, release_url);
```
This function never supported a third parameter, it was already wrong before this PR. Since the animate parameter is also gone for `insertLink` now, it should also be removed from all calls of that function. (I can't comment directly as you haven't touched these lines.)